helm/scylla-operator/templates/validatingwebhook.yaml

```diff
@@ -34,3 +34,4 @@ webhooks:
     - UPDATE
     resources:
     - nodeconfigs
+    - scylladbdatacenters
```

Comment:
which e2e test makes sure the webhook is actually correctly setup? I don't see any webhook tests for ScyllaDBDatacenter (at least in the folder structure)

Reply:
added e2e